styles.css

```diff
@@ -13,10 +13,43 @@
     align-items: flex-end;
   }
 
+  .burger {
+    padding: 1rem;
+  }
+
+  .Menu.active {
+    position: absolute;
+    display: inline;
+    width: 100%;
+    height: 100%;
+    background-color: white;
+    text-align: center;
+  }
+
+  /* Use "active class from javaScript */
+
+  .Menu.active .burger {
+    position: absolute;
+    right: 0;
+  }
+
   .portfolio-links {
     display: none;
   }
 
+  .Menu.active .portfolio-links {
+    display: block;
+    line-height: 48px;
+    font-size: 1rem;
+    list-style: none;
+    margin-top: 9rem;
+  }
+
+  .Menu.active .portfolio-links a {
+    text-decoration: none;
+    color: black;
+  }
+
```

Comment:
- [ ] Great job so far. However, there're few things you still need to work on before this project could be approved. It is required to stick to the Figma design as close as possible. This includes typography(font, font-size, font-color, font-weight), background-color, margin, padding, background-image among others. 
- The background image on your design does not tally with the background image on the figma.
- You also need to increase the font-size of the menu text in order to match the Figma design. Kindly attend to these requests.

### Your Design
![your-menu](https://user-images.githubusercontent.com/26597432/232767522-b69039a6-e601-4982-8aae-6d48444bca90.png)

### Figma Design
![temp-menu](https://user-images.githubusercontent.com/26597432/232767550-c6f6550a-1640-4b5a-ba52-798047f0561a.png)